tools-local/Microsoft.ML.AutoML.SourceGenerator/Microsoft.ML.AutoML.SourceGenerator.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Do you need this to make a nuget package everytime it builds? I thought we didn't want this shipped anywhere? Or does it have to be a nuget anyways?

Reply:
Not really, I can disable that

Reply:
## Resolved